src/node/miner.h

```diff
@@ -171,6 +172,8 @@ class BlockAssembler
 
 private:
     const Options m_options;
+    const bool m_print_modified_fee = gArgs.GetBoolArg("-printpriority", DEFAULT_PRINTPRIORITY);
```

Comment:
By doing this, you include `common/args.h` in all files that include `node/miner.h`. It is better to add the field to the `BlockAssembler::Options` struct.

Reply:
Thanks a lot, it's so much better this way!